### PR TITLE
Create advertising.tag

### PR DESCRIPTION
**Pull request by Greaper88**
Since some people are looking for a place to advertise since we locked down the multiplayer channel. What are you guys' thoughts about this just as a temporary thing?

### CODE REVIEW

File: src/main/resources/tags/info/advertising.tag [hunk 1]
```diff
@@ -0,0 +1,10 @@
+type: text
+aliases: advert, advertise
+
+---
+
+We no longer allow advretising in <#754458074818805811>, however, there are two other option available to you:
```
Comment (Legit4K) [inline src/main/resources/tags/info/advertising.tag:6]:
```suggestion
We no longer allow advertising in <#754458074818805811>, however, there are two other option available to you:
```

**Review by Legit4K — COMMENTED** [review]
Small typo on line 6

File: src/main/resources/tags/info/advertising.tag (hunk 1 above)
Comment (Greaper88) [reply src/main/resources/tags/info/advertising.tag]: ah yes. i though i checked over it all... oops. For some reason the spelling check built into my browser doesn't work in those data entry boxes on github

**Comment by lukeeey** [comment]
I don’t think we need this, plus i don’t like the idea of advertising a third party server list site.

**Comment by Chew** [comment]
As stated in the #multiplayer announcement, we do have something planned and are working hard to get it out soon! Speaking of, I think the message in #multiplayer is sufficient. We apologize for the delay 😔

**Comment by Greaper88** [comment]
Deemed unecessary, closing.

**Pull request closed without merging**